src/md/enc/metamodelrw.cpp

```diff
@@ -1433,12 +1433,9 @@ CMiniMdRW::InitOnMem(
                     {   // remember it's size.
                         iSize = m_TableDefs[i].m_pColDefs[iCol].m_cbColumn;
                     }
-                    else
-                    {   // Not first non-fixed size, so if a different size...
```

Comment:
Why drop this comment?

Reply:
The change in this file except the very first one do not look like an improvement.